src/collection/collection.service.ts

```diff
@@ -425,6 +427,27 @@ export class CollectionService {
 				);
 			}
 
+			// Update loms
+			try {
+				await this.deleteCollectionLomLinks(newCollection.id as string);
```

Comment:
on the backend i was able to avoid this one, by setting the `cleanedCollection.loms = []` higher up in the code
maybe this trick is also possible in the frontend?